nuxeo-compound-documents/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -5,4 +5,5 @@ Bundle-SymbolicName: org.nuxeo.compound.documents;singleton:=true
 Bundle-Vendor: Nuxeo
 Nuxeo-Component: OSGI-INF/core-types-contrib.xml,
  OSGI-INF/filemanager-contrib.xml,
- OSGI-INF/service-contrib.xml
+ OSGI-INF/service-contrib.xml,
+ OSGI-INF/scripted-operations-contrib.xml
```

Comment:
I would say that `operations-contrib.xml` is enough, any thoughts @troger?

Reply:
squashing all the contribs?

Reply:
No, the name of the contrib.